x-pack/plugins/ml/common/types/ml_url_generator.ts

```diff
@@ -5,27 +5,21 @@
  */
 
 import { RefreshInterval, TimeRange } from '../../../../../src/plugins/data/common/query';
-import { JobId } from '../../../reporting/common/types';
+import { JobId } from './anomaly_detection_jobs/job';
 import { ML_PAGES } from '../constants/ml_url_generator';
+import { DataFrameAnalyticsType } from './data_frame_analytics';
 
 type OptionalPageState = object | undefined;
 
 export type MLPageState<PageType, PageState> = PageState extends OptionalPageState
-  ? { page: PageType; pageState?: PageState }
+  ? { page: PageType; pageState?: PageState; excludeBasePath?: boolean }
   : PageState extends object
-  ? { page: PageType; pageState: PageState }
+  ? { page: PageType; pageState: PageState; excludeBasePath?: boolean }
   : { page: PageType };
```

Comment:
I guess `excludeBasePath` is applicable for any page, isn't it?

Reply:
You're right. Updated here https://github.com/elastic/kibana/pull/76735/commits/cea397ee78ebf873893a6bc05185ddacaa40704c